cmd/arena/commands/gpu_info.go

```diff
@@ -0,0 +1,212 @@
+package commands
+
+import (
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"encoding/json"
+	"fmt"
+	log "github.com/sirupsen/logrus"
+	v12 "k8s.io/api/core/v1"
```

Comment:
Please change it to `corev1 "k8s.io/api/core/v1"` , and `metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"`